social-network/src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import './App.css';
+import Creativityarticle from './creatify';
+import Missedarticle from './creatify/missed-article.js';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <h3>For you</h3>
+        <Creativityarticle />
```

Comment:
Creativityarticle as a component would've made sense, but not Creativityarticles which you have here.   Same for missed articles.